LandSandBoat/server
# Rename the `update` status.

**_I affirm:_**

## What does this pull request do?
The update state is primarily *used by* mobs, but is not mob exclusive.

See `CZoneEntities::SpawnNPCs()` for an example of it being used on non-mobs.

## Steps to test these changes
see it build. log in and observe nothing has change other than the human readable identifier.
